ksemmodbus/__init__.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
+#########################################################################
+#  Copyright 2019 Thomas Hengsberg <thomas@thomash.eu>
+#########################################################################
+#  This file is part of SmartHomeNG.   
+#
+#  Sample plugin for new plugins to run with SmartHomeNG version 1.4 and
+#  upwards.
+#
+#  SmartHomeNG is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SmartHomeNG is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SmartHomeNG. If not, see <http://www.gnu.org/licenses/>.
+#
+#########################################################################
+
+from lib.model.smartplugin import *
+
+from .ksem import Ksem
+
+class Kostalmodbus(SmartPlugin):
```

Comment:
Maybe you should have another class name?

Reply:
Of course. Changed class name and tweaked the readme